pandas/tests/test_downstream.py

```diff
@@ -104,13 +110,17 @@ def test_pandas_datareader():
         'F', 'quandl', '2017-01-01', '2017-02-01')
 
 
+# importing from pandas, Cython mport warning
```

Comment:
Yes (later ones I added have a reference to the issue).

> Should we have an issue to make sure to remove those filters again at some point?

Maybe, but I think that's largely out of our control. IIUC, it would require a release from each of those libraries with their extensions compiled by a new enough Cython.

Reply:
> we should not generally ignore all warnings here

I'm restricting these now, but we may want to ignore all these warnings. Fixing e.g. geopandas required

1. ignore Cython warning
2. ignore geopandas importing pandas.util.decorators (shall  I open an issue?)
3. ignore fiona using `Logger.warn` instead of `Logger.warning`

It seems like we're exposing ourself to a lot of spurious failures if we don't filter these heavily.